windows/bleachbit_nsis_multiuser.pot

```diff
@@ -0,0 +1,93 @@
+# English translation of bleachbit_nsis_multiuser.pot
+# Copyright (C) 2019 Andrew Ziem
```

Comment:
This file looks like OK, but to work with Launchpad, it needs to be integrated into bleachbit.pot using in `po/Makefile`. Look at the Makefile to see how it combines `cleanerml.pot` with `desktop.pot`. If you need help, I can do it.

Reply:
OK, I also add this comment to issue #529...